Fixed #34688 -- Removed contrib.sitemaps.ping_google() and ping_google management command. (#17039)
django/django: docs/releases/5.0.txt

* Support for GEOS 3.6 and 3.7 is removed.

:mod:`django.contrib.sitemaps`
------------------------------

* The ``django.contrib.sitemaps.ping_google()`` method and the ``ping_google``

>   management command have been removed due to a deactivation of the Sitemaps

Reviewer: ```suggestion
  management command are removed due to a deactivation of the Sitemaps
```

* Support for calling ``format_html()`` without passing args or kwargs will be
  removed.

> * The ``django.contrib.sitemaps.ping_google()`` method and the ``ping_google``
>   management command are deprecated.

Reviewer 2: ```suggestion
* The ``django.contrib.sitemaps.ping_google()`` method and the ``ping_google``
  management command are deprecated and, starting on 5.0, they will be noop
  until being removed.
```

* Passing ``nulls_first=False`` or ``nulls_last=False`` to ``Expression.asc()``
  and ``Expression.desc()`` methods, and the ``OrderBy`` expression is no
  longer allowed.

> Features removed without deprecation.

Reviewer: There is no need for a separate header. Features removed without a deprecation should be described in the [Backwards incompatible changes in 5.0](https://docs.djangoproject.com/en/dev/releases/5.0/#backwards-incompatible-changes-in-5-0).
